src/C1_verification.py

```diff
@@ -7,19 +17,22 @@
 
 def lookup_file(file_path, name):
     """
+    Checks whether file specified in `file_path` exists.
 
-    :param file_path:
-    :param name:
+    If it does not exist a critical logging message and a system exit is risen.
```

Comment:
```suggestion
    If a critical logging message does not exist and a system exit is raised.
```

Reply:
It should be "If it does not exist, a FileNotFoundError is raised."